client/src/routes/layout.js

```diff
@@ -6,16 +6,19 @@ import { CCF_THEME } from "../actions/theme";
 import WelcomeHeader from "../component/welcomeHeader/welcomeHeader";
 import { Outlet } from "react-router-dom";
 import LinkSection from "../component/linkSection/linkSection";
+import { GoogleOAuthProvider } from '@react-oauth/google';
 
 const Layout = () => {
   return (
-    <ThemeProvider theme={CCF_THEME}>
-      <LinkSection />
-      <Box padding="1rem" margin="4rem 0.5rem 1rem 0.5rem">
-        <WelcomeHeader />
-        <Outlet />
-      </Box>
-    </ThemeProvider>
+    <GoogleOAuthProvider clientId="858566898033-jau549hlkcrroj0fhgg0upj53anhbcde.apps.googleusercontent.com">
```

Comment:
is it fine to just have this in code? should we put it in .env..? idk

Reply:
just did! good catch i forgot abt that